dags/hindawi/parser.py

```diff
@@ -0,0 +1,200 @@
+                prefixes=self.prefixes,
+                extra_function=lambda x: x,
+            ),
+            TextExtractor(
+                destination="journal_volume",
+                source="ns0:metadata/ns1:record/ns0:datafield/[@tag='773']/ns0:subfield/[@code='v']",
+                prefixes=self.prefixes,
+                extra_function=lambda x: x,
+            ),
+            CustomExtractor(
+                destination="arxiv_eprints", extraction_function=self._get_arxiv
+            ),
+            TextExtractor(
+                destination="journal_year",
+                source="ns0:metadata/ns1:record/ns0:datafield/[@tag='773']/ns0:subfield/[@code='y']",
+                prefixes=self.prefixes,
+                extra_function=lambda x: x,
+            ),
+            CustomExtractor(
+                destination="copyright_statement",
+                extraction_function=self._get_copyright_statement,
+            ),
+            CustomExtractor(
+                destination="copyright_year",
+                extraction_function=self._get_copyright_year,
+            ),
+            CustomExtractor(
+                destination="license", extraction_function=self._get_license
+            ),
+        ]
+        super().__init__(extractors)
+
+    def _get_authors(self, article: ET.Element):
+        authors_all = article.findall(
+            "ns0:metadata/ns1:record/ns0:datafield/[@tag='100']", self.prefixes
+        )
+        authors_all + (
+            article.findall(
+                "ns0:metadata/ns1:record/ns0:datafield/[@tag='700']", self.prefixes
+            )
+        )
+        authors = []
+        for author in authors_all:
+            author_ = {
+                "raw_name": author.find(
+                    "ns0:subfield[@code='a']", self.prefixes
+                ).text.replace("\n", ""),
+                "affiliations": self._get_affiliations(author),
+            }
+            orcid = author.find("ns0:subfield[@code='j']", self.prefixes)
+            if orcid.text:
+                author_["orcid"] = orcid.text
+            authors.append(author_)
+
+        return authors
+
+    def _get_affiliations(self, author):
+        affiliations = [
+            affiliation.text.replace("\n", "")
+            for affiliation in author.findall("ns0:subfield[@code='u']", self.prefixes)
+        ]
+        parsed_affiliations = [
+            {
+                "value": affiliation,
+                "organization": (",").join(affiliation.split(",")[:-1]),
+                "country": affiliation.split(", ")[-1:][0],
+            }
+            for affiliation in affiliations
+        ]
+        return parsed_affiliations
+
+    def _get_arxiv(self, article: ET.Element):
+        arxiv = article.find(
+            "ns0:metadata/ns1:record/ns0:datafield/[@tag='037']/ns0:subfield/[@code='9']",
+            self.prefixes,
+        )
+        if arxiv is None:
```

Comment:
```suggestion
        if not arxiv:
```

Reply:
I cannot do it, because arxiv Element object. For some reason:
`if not arxiv` returns `True`
and `if arxiv is None:` returns False

I understand that `not` should return True if the variable is not assigned, but in this case, we have the Element object assigned here
And `is None` when is None